### ISSUE TITLE
Unable to install on Obsidian Mobile

### ISSUE THREAD

**Issue by Dennyyusuf** [cumany/obsidian-floating-toc-plugin#26]
Hi Developer!

Thanks for this great plugin,
I have successfully installed the PC version and wondering if it is also possible install it on mobile?

Thanks!

**Comment by FabrizioMusacchio** [comment]
Same on the iPhone: it does not load the plugin, while on macOS everything works fine.

**Comment by erauner12** [comment]
Yeah I have same issue. I see that there is an option to make it available on mobile? But is does not seem to work
